src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -223,4 +224,11 @@ describe('NameAndPictureScreen', () => {
     fireEvent.press(getByText('skip'))
     expect(AccountActions.setName).not.toHaveBeenCalled()
   })
+
+  it('usernames appear as expected', () => {
```

Comment:
this test looks kinda circular. 

There's probably a way to write the code such that it becomes more testable. For example, a pure function `chooseRandomWord(wordList, randomSeed = 'arbitrary_seed')` could be unit tested, and then mocked to unit test `generateUsername` with more precision